### ISSUE TITLE
[CAL-4073] Kyiv timezone missing

### ISSUE THREAD

**Issue by rostyc** [calcom/cal.com#12224]
When I try to set my timezone to Europe/Kyiv I get “no options” error

![IMG_2471](https://github.com/calcom/cal.com/assets/54397912/a86f97e8-6ece-425c-94d1-1ebdd5d234c4)

<sub>[CAL-4073](https://linear.app/calcom/issue/CAL-4073/kyiv-timezone-missing)</sub>

**Comment by skverma618** [comment]
@rostyc can you provide the steps to recreate this

**Comment by rostyc** [comment]
@skverma618 in account settings (https://app.cal.com/settings/my-account/general) there is a field for setting up your timezone, in that list my timezone (Europe/Kyiv) is nowhere to be found

**Comment by PeerRich** [comment]
![CleanShot 2023-11-09 at 16 52 46@2x](https://github.com/calcom/cal.com/assets/8019099/055d7b19-3dc6-4ff6-91f1-d9dcfb5e848e)
all timezones are written in English according to official timezone standards

**Comment by rostyc** [comment]
@PeerRich no, you’re wrong. Europe/Kyiv is the timezone name since tz version 2022b

**Comment by PeerRich** [comment]
city-timezones has been updated: https://github.com/kevinroberts/city-timezones/pull/26

and our dependency PR is here: https://github.com/calcom/cal.com/pull/15927

**Comment by CarinaWolli** [comment]
<img width="320" alt="Image" src="https://github.com/user-attachments/assets/32d2d0b2-6273-434e-8c67-95acb63435e8" />
